syntax = "proto3";

package image_search;

service ImageSearchService {
    rpc SearchImage (SearchRequest) returns (ImageResponse);
}

message SearchRequest {
    string keyword = 1;
}

message ImageResponse {
    bytes image_data = 1;
}
